django-backend/fecfiler/committee_accounts/views.py

```diff
@@ -1,15 +1,58 @@
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from .models import CommitteeAccount
+from .serializers import CommitteeAccountSerializer, CommitteeMemberSerializer
+from fecfiler.settings import FFAPI_COMMITTEE_UUID_COOKIE_NAME, FFAPI_COOKIE_DOMAIN
 import logging
 
 logger = logging.getLogger(__name__)
 
 
+class CommitteeViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+    serializer_class = CommitteeAccountSerializer
+
+    def get_queryset(self):
+        user = self.request.user
+        return CommitteeAccount.objects.filter(members=user)
+
+    @action(detail=True, methods=["get"])
+    def users(self, request, pk):
+        committee = self.get_object()
+        serializer_context = {"committee_id": committee.id}
+        queryset = self.filter_queryset(committee.members.all())
+        page = self.paginate_queryset(queryset)
+        if page is not None:
+            serializer = CommitteeMemberSerializer(
+                page, many=True, context=serializer_context
+            )
+            return self.get_paginated_response(serializer.data)
+
+        serializer = CommitteeMemberSerializer(
+            queryset, many=True, context=serializer_context
+        )
+        return Response(serializer.data)
+
+    @action(detail=True, methods=["post"])
+    def activate_committee(self, request):
+        committee = self.get_object()
```

Comment:
probably obvious but we'll need to add security checks here to make sure a user has access to the committee account

Reply:
yeah the queryset handles that because it will only return committees that the user is part of